2/contrib/jenkins/install-plugins.sh

```diff
@@ -225,8 +225,14 @@ function doDownload() {
         url="$JENKINS_UC_DOWNLOAD/plugins/$plugin/$version/${plugin}.hpi"
     fi
 
+    JENKINS_UC_INSECURE=${JENKINS_UC_INSECURE:-"false"}
+    if [[ -n "$JENKINS_UC_INSECURE" ]] && [[ "$JENKINS_UC_INSECURE" = true || "$JENKINS_UC_INSECURE" = 1 ]]; then
```

Comment:
why support both true and 1 ... with other env vars, we employed a patter that if
- the env was not empty
- and it was set to something other than false
toggle the behavior

Reply:
Resolved. I made the test syntax a bit verbose to protect unset variables checking in shell, but it now uses the logic you described.